lib/tapi_ta_events/tapi_ta_events.c

```diff
@@ -0,0 +1,104 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/** @file
+ * @brief API for TA events
+ *
+ * Copyright (C) 2024-2024 OKTET Labs Ltd. All rights reserved.
```

Comment:
Shouldn't the copyright be actually for ARK Networks, not OKTET Labs?

Reply:
Sure, fixed.